external/dynamo/generalized/frontend.py

```
@@ -0,0 +1,434 @@
+
+        # Initialize TPS CSV and start background writer
+        try:
+            csv_dir = os.path.dirname(self._tps_csv_path)
+            if csv_dir and not os.path.exists(csv_dir):
+                os.makedirs(csv_dir, exist_ok=True)
+            if not os.path.exists(self._tps_csv_path):
+                async with self._tps_lock:
+                    with open(self._tps_csv_path, "w", newline="") as f:
+                        writer = csv.writer(f)
+                        writer.writerow(["ts_epoch_ms", "requests", "interval_s", "req_per_sec"])
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)